packages/main/src/Popover.js

```diff
@@ -336,17 +337,21 @@ class Popover extends Popup {
 		this.show();
 	}
 
-	show() {
+	async show() {
 		let placement;
-		const popoverSize = this.popoverSize;
-		const openerRect = this._opener.getBoundingClientRect();
+		const popoverSize = await this._popoverSize;
 
-		if (this.shouldCloseDueToNoOpener(openerRect) && this.isFocusWithin()) {
+		if (this.isOpen()) {
+			// update opener rect if it was changed during the popover is opened
```

Comment:
// update opener rect if it was changed during the popover **being** opened

Reply:
done